docker-compose.yml

```diff
@@ -322,6 +325,9 @@ services:
   ingest-consumer:
     <<: *sentry_defaults
     command: run ingest-consumer --all-consumer-types
+  ingest-replay-recordings:
+    <<: *sentry_defaults
+    command: run ingest-replay-recordings
```

Comment:
Dang 33 services 😭

```
clickhouse
cron
geoipupdate
ingest-consumer
ingest-replay-recordings
kafka
memcached
nginx
post-process-forwarder-errors
post-process-forwarder-transactions
postgres
redis
relay
sentry-cleanup
smtp
snuba-api
snuba-cleanup
snuba-consumer
snuba-outcomes-consumer
snuba-replacer
snuba-replays-consumer
snuba-sessions-consumer
snuba-subscription-consumer-events
snuba-subscription-consumer-transactions
snuba-transactions-cleanup
snuba-transactions-consumer
subscription-consumer-events
subscription-consumer-transactions
symbolicator
symbolicator-cleanup
web
worker
zookeeper
```

Reply:
11 snubas 😖

Reply:
https://github.com/getsentry/snuba/issues/1670

That said this would only help with a 33% reduction.

Looks like we need a similar issue for Sentry to consolidate all those `query-subscription-consumer` and `post-process-forwarder` services.